specification/eventgrid/Azure.Messaging.EventGrid/examples/2023-06-01-preview/PublishExample.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
have messaged service team about getting examples for operations -- made this one in the meantime